aspnetcore/fundamentals/middleware/rate-limit/samples/WebRateLimitAuth/Program.cs

```diff
@@ -0,0 +1,197 @@
+#define ADMIN // FIRST ADMIN
```

Comment:
@Kahbazi @bugproof @pentp please review rate limiting. Do we need more samples? I think I'll leave the [custom rate limiter](https://github.com/Rick-Anderson/WebRateLimitMw/blob/main/MyCustomLimiter.cs) for another PR. Most folks won't need to implement `RateLimiter`

Reply:
Seems fine for a start.